utils/data_receiver/data_receiver.pony

```diff
@@ -18,15 +18,53 @@ Copyright 2017 The Wallaroo Authors.
 
 use "net"
 use "wallaroo_labs/bytes"
+use "wallaroo_labs/logging"
 use "wallaroo_labs/options"
 
+use @logf[I32](severity: U8, category: U8, fmt: Pointer[U8] tag, ...)
+
 actor Main
   new create(env: Env) =>
     var required_args_are_present = true
     var l_arg: (Array[String] | None) = None
     var output_mode: OutputMode = Write
     var input_mode: InputMode = Streaming
 
+    // let Log.emerg = U8(0)
```

Comment:
I like all of this, but I think it's worth moving it to a unit test/e2e test thing.

Reply:
Agreed, will do for a final'ish PR